modules/getting-started/proc-customize-rhdh-homepage.adoc

```diff
@@ -1,11 +1,10 @@
 [id='proc-customize-rhdh-homepage_{context}']
 = Customizing the Home page in {product}
 
-In {product}, the Home page data is configurable, which can be passed into the `app-config.yaml` file as a proxy. You can provide the Home page data using the following ways:
+To access the Home page in {product}, the base URL must include the `/developer-hub` proxy. You can configure the Home page by passing the data into the `app-config.yaml` file as a proxy. You can provide data to the Home page by using the following sources:
```

Comment:
Home page can be accessed without any configuration with static data i.e configuration is not a necessity. However If the user configures with a proxy then the data powering the home page can be updated

Reply:
@invincibleJai I only rearranged existing content here -- the bit about the base URL was mentioned at the very end of the Home page doc but at the very beginning of the Tech Radar doc, so I moved it to the beginning of the Home page doc, but did not change the meaning of the text. Is there something here that is / seems technically inaccurate and needs to be changed or removed?